lib/network/modules/InteractionHandler.js

```diff
@@ -165,8 +165,11 @@ class InteractionHandler {
 
   onContext(event) {
     let pointer = this.getPointer({x:event.clientX, y:event.clientY});
-    this.checkSelectionChanges(pointer, event);
-    this.selectionHandler._generateClickEvent('oncontext', event, pointer);
+    const hoveredElements = {
+      nodes: this.selectionHandler.getNodeAt(pointer.DOM),
+      edges: this.selectionHandler.getEdgeAt(pointer.DOM)
+    }
+    this.selectionHandler._generateClickEvent('oncontext', Object.assign({}, event, hoveredElements), pointer);
```

Comment:
Personal note: that's a pretty nifty use of `Object.assign()`. Wouldn't have thought of it myself, TIL.